Formula/needle.rb

```diff
@@ -12,11 +12,13 @@ class Needle < Formula
     sha256 "ea9e38d739acc83ca9a58ba10aef72456e73f22e790462f243192db7aa0a814d" => :mojave
   end
 
-  depends_on xcode: ["11.3", :build]
+  depends_on xcode: ["11.6", :build]
   depends_on xcode: "6.0"
 
   def install
     system "make", "install", "BINARY_FOLDER_PREFIX=#{prefix}"
+    bin.install "./Generator/bin/needle"
+    libexec.install "./Generator/bin/lib_InternalSwiftSyntaxParser.dylib"
```

Comment:
The library doesn't need to be next to the executable to function?

Reply:
Nope, I've configured it in `make install` so that it could find the .dylib in relative path.